armi/physics/fuelCycle/translationFunctions.py

```diff
@@ -0,0 +1,690 @@
+# Copyright 2022 TerraPower, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
Can we get a big docstring here saying why this file exists?   For a few reasons:

1. It's the standard. Look at the top of `fuelHandlers.py` for an example. But the idea is just to explain to people why this file exists.
2. I don't fully understand why we split `fuelHandlers.py` into that and `translationFunctions.py`.  If it's just for convenience, cool. But I should probably know.
3. If I don't understand it _now_, what are the changes that someone else will understand why this file exists 4 years from now?

Reply:
Added a dosctring. Let me know if it needs more. The purpose of splitting up `fuelHandlers.py` is to reduce the size of the file for convenience. And thank you for your feedback! I will keep that in mind for future PR's.